cms/djangoapps/contentstore/helpers.py

```diff
@@ -0,0 +1,286 @@
+"""
+Helper methods for Studio views.
+"""
+
+import urllib
+from lxml import etree
+
+from django.utils.translation import gettext as _
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.locator import DefinitionLocator, LocalId
+from xblock.core import XBlock
+from xblock.fields import ScopeIds
+from xblock.runtime import IdGenerator
+from xmodule.modulestore.django import modulestore
+
+# from cms.djangoapps.contentstore.views.preview import _load_preview_block
+from cms.djangoapps.models.settings.course_grading import CourseGradingModel
+from common.djangoapps.student import auth
+from common.djangoapps.student.roles import CourseCreatorRole, OrgContentCreatorRole
+
+try:
+    # Technically this is a django app plugin, so we should not error if it's not installed:
+    import openedx.core.djangoapps.content_staging.api as content_staging_api
+except ImportError:
+    content_staging_api = None
```

Comment:
Can you help me understand this? The module [`content_staging.api`](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_staging/api.py) exists on master in this repo, so an `ImportError` will never be raised, right?

Reply:
I have no idea. This code is old and just moved verbatim from the contentstore/views/helpers.py file. Since I assume there is or there was a reason for this, and I have not touched this, I would say changing anything here is at least out of scope (would just introduce risk.)

Reply:
@jesperhodge Ah, given that it is existing code, no worries 👍🏻 

(I mean, it's still not great, but it's not your problem :-)